src/textures/Texture.js

```diff
@@ -225,6 +185,46 @@ Texture.prototype = Object.assign( Object.create( EventDispatcher.prototype ), {
 
 	},
 
+	_getDataURL: function ( image ) {
```

Comment:
Maybe we could bring back `ImageUtils` for this`? `ImageUtils.getDataURL` seems more appropriate.